lexicons/com/atproto/label/query.json

```diff
@@ -0,0 +1,42 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.label.query",
```

Comment:
It may be more consistent with the style post-lex refactor to use something verb-noun like `queryLabels` here.